docs/testingplatform/architecture.md

```diff
@@ -32,23 +37,23 @@ Copyright(c) Microsoft Corporation.  All rights reserved.
 Passed! - Failed: 0, Passed: 1, Skipped: 0, Total: 1, Duration: 5ms - Contoso.UnitTests.dll (win-x64 - .NET 9.0.0-preview.1.24080.9)
 ```
 
-When working in the testing space, it is common to collect information from test sessions to evaluate code quality.
```

Comment:
This is interesting detail, but it seemed to distract from what you were trying to highlight, which was starting one more process.